### PR TITLE
chore: cherry pick translation text zh-cn

### PR DESCRIPTION
**Pull request by Helias**
cherry pick from https://github.com/azerothcore/aowow/pull/36

### CODE REVIEW

File: localization/locale_zhcn.php [hunk 1]
```diff
@@ -137,24 +137,24 @@
...
+        'cannotComment' => "[你已被禁止发表评论。]",
+        'textLength'    => "[你的评论有 %d 个字符，必须至少有 %d 个字符，最多 %d 个字符。]",
 
         'moreTitles'    => array(
             'reputation'    => "网站声望",
             'whats-new'     => "新内容",
             'searchbox'     => "搜索框",
             'tooltips'      => "工具提示",
             'faq'           => "常见问答",
-            'aboutus'       => "什么是AoWoW？",
```
Comment (Sarjuuk) [inline localization/locale_zhcn.php:150]: Not sure this should be translated, but i don't care either way.

File: localization/locale_zhcn.php [hunk 2]
```diff
@@ -137,24 +137,24 @@
         'bannedRating'  => "你评级评论的权力已被冻结。", # LANG.tooltip_banned_rating
         'tooManyVotes'  => "你已经达到每日投票上限。请明天再来！", # LANG.tooltip_too_many_votes
         'alreadyReport' => "您已报告。", # LANG.ct_resp_error7
-        'textTooShort'  => "[Your message is too short.]",
-        'cannotComment' => "[You have been banned from writing comments.]",
-        'textLength'    => "[Your comment has %d characters and must have at least %d and at most %d characters.]",
+        'textTooShort'  => "[你的留言太短了。]",
```
Comment (Sarjuuk) [inline localization/locale_zhcn.php:140]:
```suggestion
        'textTooShort'  => "你的留言太短了。",
```
Square brackets usually denote missing translations. They should be removed.

File: localization/locale_zhcn.php [hunk 3]
```diff
@@ -1971,9 +1971,7 @@
             "每5秒恢复%d点生命值。",
             "法术穿透提高%d。",
             "使你的盾牌格挡值提高%d。",
-            "Unknown Bonus #%d (%d)",
+            "未知加成 #%d (%d)",
         )
     )
 );
-
-?>
```
Comment (Sarjuuk) [inline localization/locale_zhcn.php:1979]: uhh .. that closing tag is needed.

**Review by Sarjuuk — CHANGES_REQUESTED** [review]
Hi, thanks for the translation.

There are one or two things that should be fixed before i merge this.
I highly recommend to also apply them on your end.

File: localization/locale_zhcn.php (hunk 3 above)
Comment (Sarjuuk) [reply localization/locale_zhcn.php:1979]: Oh, that was introduced by the auto merge.

..wat?

Comment (Helias) [reply localization/locale_zhcn.php]: fixed all, thanks

File: localization/locale_zhcn.php (hunk 2 above)
Comment (Helias) [reply localization/locale_zhcn.php]: fixed

**Comment by Sarjuuk** [comment]
tyvm :]

**Comment by Sarjuuk** [comment]
Please ignore the dangling revert. I fatfingered my phone >_>

**Comment by Helias** [comment]
thank you for maintaining this project ;-)

**Pull request merged**